cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -560,8 +562,11 @@ def test_custom_button_dialog_service_archived():
     pass
 
 
-@pytest.mark.manual
-def test_custom_button_dialog(button_group):
+@pytest.mark.parametrize("context", [ViaUI, ViaSSUI])
+@pytest.mark.uncollectif(
+    lambda context, button_group: context == ViaSSUI and "GENERIC" in button_group
```

Comment:
Can you comment why SSUI and Generic is uncollected .

Reply:
Sure, thanks @sshveta 
`Generic object` on `SSUI` as resource; Unable to get `All` or `Details` page of generic object on SSUI. Custom button display  depends on `All` and `Details` page so `Generic object custom button not supported by SSUI`. Added reason :+1: